test/fixtures/for/in-auto-const/actual.js

```diff
@@ -1 +1 @@
-for k in {}: k
+for const k in {}: k
```

Comment:
let's be sure to add tests for `now` and `for k in {}`, the latter raising an error. 

(this test itself can either be deleted or renamed)